.github/CHANGELOG.md

```diff
@@ -2,6 +2,30 @@
 
 <h3>New features since last release</h3>
 
+* A new  `qml.make_drawing` method was added. This allows for qnodes to easily be drawn by simply giving an example input.
+  
+  Here is some example usage:
+
+  ```python
+
+  qml.enable_tape()
+
+  @qml.qnode(dev)
+  def circuit(a, w):
+      qml.Hadamard(0)
+      qml.CRX(a, wires=[0, 1])
+      qml.Rot(*w, wires=[1])
+      qml.CRX(-a, wires=[0, 1])
+      return qml.expval(qml.PauliZ(0) @ qml.PauliZ(1))
+
+  result = qml.make_drawing(circuit)(a=2.3, w=[1.2, 3.2, 0.7])
+  print(result)
+  #  0: ──H──╭C────────────────────────────╭C─────────╭┤ ⟨Z ⊗ Z⟩
+  #  1: ─────╰RX(2.3)──Rot(1.2, 3.2, 0.7)──╰RX(-2.3)──╰┤ ⟨Z ⊗ Z⟩
```

Comment:
The double brakets can be very confusing... would it be neater to do something like:

``` pycon
  draw = qml.make_drawing(circuit)
  print(draw(a=2.3, w=[1.2, 3.2, 0.7]))
```

Or even to change the function so that `draw` would print automatically?

Reply:
Do we want drawer to print automatically or return the string? I have no preference.

Reply:
I have a slight preference for leaving it as a string, since this is more flexible (a user can determine exactly how they want to use it, rather than being forced to use `stdout`)

Reply:
Ok, convinced!